apps/api/src/app/widgets/usecases/mark-message-as/mark-message-as.usecase.ts

```diff
@@ -87,18 +82,15 @@ export class MarkMessageAs {
     }
   }
 
-  private updateSocketCount(subscriber: SubscriberEntity, count: number, mark: string) {
-    const eventMessage = `un${mark}_count_changed`;
```

Comment:
this one is very cute (made by George from the past 🤦‍♂️), but this event is more difficult to debug/discover in the code like this.